LWOTC_WeaponPack/Config/XComLW_WeaponPack.ini

```diff
@@ -90,71 +82,50 @@ SniperRifle_COIL_SCHEMATIC_SUPPLYCOST=125
 SniperRifle_COIL_SCHEMATIC_ALLOYCOST=15
 SniperRifle_COIL_SCHEMATIC_ELERIUMCOST=5
 
+MarksmanRifle_COIL_SCHEMATIC_SUPPLYCOST=125
+MarksmanRifle_COIL_SCHEMATIC_ALLOYCOST=15
+MarksmanRifle_COIL_SCHEMATIC_ELERIUMCOST=5
+
 Pistol_COIL_SCHEMATIC_SUPPLYCOST=40
 Pistol_COIL_SCHEMATIC_ALLOYCOST=10
 Pistol_COIL_SCHEMATIC_ELERIUMCOST=5
 
-[LWOTC_WeaponPack.X2Item_LWBeamSchematics]
-SMG_BEAM_SCHEMATIC_SUPPLYCOST = 250
-SMG_BEAM_SCHEMATIC_ALLOYCOST = 20
-SMG_BEAM_SCHEMATIC_ELERIUMCOST = 10
-; TODO
-
-[LWOTC_WeaponPack.X2Item_LWConventionalWeapons]
-SMG_CONVENTIONAL_BASEDAMAGE=	(Damage=3, Spread=1, PlusOne=0, Crit=2, Pierce=0, Shred=0, Tag="", DamageType="Projectile_Conventional")
-SMG_CONVENTIONAL_AIM = 0
-SMG_CONVENTIONAL_CRITCHANCE = 0
-SMG_CONVENTIONAL_ICLIPSIZE = 3
-SMG_CONVENTIONAL_ISOUNDRANGE = 27
-SMG_CONVENTIONAL_IENVIRONMENTDAMAGE = 5
-SMG_CONVENTIONAL_UPGRADESLOTS = 1
-
-; **** WEAPON RANGE TABLES ****
-; MIDSHORT WEAPONS: SMG
```

Comment:
Will this table override the ini config values? or will the ini config values override this table?